pai-management/src/frameworklauncher/dockerfile

```diff
@@ -19,6 +19,7 @@ FROM hadoop-run
 
 RUN apt-get -y update && \
     apt-get -y install maven
+RUN rm -rf /var/lib/apt/lists
```

Comment:
Why do you need this line here even when hadoop-run image has already included it? And why do you delete the whole directory (rm -rf .../lists) here, while you keep the directory itself (rm -rf .../list/*****) in hadoop-run?

Reply:
Well, clean the apt-get could reduce the image size.

Reply:
@hwuu ,
Good point, I haven't noticed this difference yet.
I will update it to (rm -rf .../list/*) in both.
It's OK to remove the whole folder if you won't use apt-get again, but maybe there is difference.
Because 'hadoop-run' is base, can we can build image from it, so I believe the (rm -rf .../list/*) is more safe.